src/lib/comms/include/sol-netctl.h

```diff
@@ -309,6 +375,146 @@ int32_t sol_netctl_service_get_strength(
     const struct sol_netctl_service *service);
 
 /**
+ * @brief Gets the service favorite
+ *
+ * This function gets the service favorite for the netctl service,
+ * after the service is to give by service monitor.
+ *
+ * @param service The service structure which the service favorite is
+ * desired
+ *
+ * @return 0-100 on success, -errno on failure.
+ */
+bool sol_netctl_service_get_favorite(
+    const struct sol_netctl_service *service);
+
+/**
+ * @brief Gets the service immutable
+ *
+ * This function gets the service immutable for the netctl service,
+ * after the service is to give by service monitor.
+ *
+ * @param service The service structure which the service immutable is
+ * desired
+ *
+ * @return 0-100 on success, -errno on failure.
+ */
+bool sol_netctl_service_get_immutable(
```

Comment:
All the same comments that I did about sol_netctl_service_get_favorite() applies to this function